Add some headline notes (#6)
ACCESS-NRI/cookiecutter-pypackage-access: README.md

Work in progress: please report any issues you encounter.

## Cliff Notes

> - All files related to our template are in the `{{cookiecutter.project_slug}}` directory.

Reviewer: ```suggestion
- All files related to our template are in the [{{cookiecutter.project_slug}}](https://github.com/ACCESS-NRI/cookiecutter-pypackage-access/tree/main/%7B%7Bcookiecutter.project_slug%7D%7D) directory.
```
I would add a link to it.

> - The `{{cookiecutter.project_slug}}` directory is the root of the package.

Reviewer: ```suggestion
- The `{{cookiecutter.project_slug}}` directory will become the root of the created repository.
```
